### ISSUE TITLE
Using custom OData service without Azure

### ISSUE THREAD

**Issue by richardalmonte** [adrianhall/develop-mobile-apps-with-csharp-and-azure#16]
Hi,

I have a question, is there a way to use Offline Sync example (chapter 3 -TaskList) with a custom (private) OData service without  Azure services? 

I have implemented and adapted the TaskList example, but now I need to use my OData service and would be nice to use a library like Microsoft.Azure.Mobile.Client SDK. 

My OData service is exposed by SharePoint and would be great if I could implement the client without another service in the middle.

Thanks

**Comment by adrianhall** [comment]
No - the OData service that is required is opinionated.  You can't just use any OData service - it must conform to the Azure Mobile Apps spec.

**Comment by ericbrunner** [comment]
@richardalmonte
Hi Richard, We I began 3 month ago I had the same thoughts to host all in-house. But Azure is very effortable and we decided to put our data needed for the mobile app that is accessed throught backend tablecontroller in an Azure SQL Server all in a S0 Standard Service Plan which is about ~ 100 USD / Month in summary . You get a lot of 99.9 % Service Level Agreement. We moved the subset of our in-house on-premises Sql Server to Azure and all works well. Hope my remarks help a bit.


best
Eric
